mappings/clacso.py

```diff
@@ -92,11 +85,19 @@ def format_has_part(self):
                         if ('.pdf' in link.get('href', [])):
                             pdf_link = link.get('href')
                             if 'bitstream/CLACSO' in pdf_link:
-                                response = requests.get(f'{self.HANDLE_URL}{pdf_link}')
+                                response = requests.get(f'{PDF_PART_URL}{pdf_link}')
                             else:
                                 response = requests.get(link.get('href'))
                             if response.status_code == 200:
-                                has_part = (f'1|{self.HANDLE_URL}{pdf_link}|clacso|application/pdf|{{"reader": false, "download": true, "catalog": false, "embed": true}}')
-                                clean_has_part.append(has_part)
-                                self.record.has_part = clean_has_part
-                                break
+                                has_part_format = (f'1|{PDF_PART_URL}{pdf_link}|clacso|application/pdf|{{"reader": false, "download": true, "catalog": false, "embed": true}}')
```

Comment:
Looks good! Feel free to use the FileFlags and also the Part classes

Reply:
Sure